packages/ios_platform_images/ios/Classes/IosPlatformImagesPlugin.swift

```diff
@@ -0,0 +1,69 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import Flutter
+import Foundation
+
+public final class IosPlatformImagesPlugin: NSObject, FlutterPlugin {
+
+  private let channel: MethodChannel
+
+  init(channel: MethodChannel) {
+    self.channel = channel
+  }
+
+  public static func register(with registrar: FlutterPluginRegistrar) {
+    let channel = FlutterMethodChannel(
+      name: "plugins.flutter.io/ios_platform_images",
+      binaryMessenger: registrar.messenger())
+
+    let instance = IosPlatformImagesPlugin(channel: channel)
+    registrar.addMethodCallDelegate(instance, channel: channel)
+  }
+
+  public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
+    switch call.method {
+    case "loadImage":
+      loadImage(call, result)
+    case "resolveURL":
+      resolveURL(call, result)
+    default:
+      result(FlutterMethodNotImplemented)
+    }
+  }
+
+  private func loadImage(_ call: FlutterMethodCall, _ result: @escaping FlutterResult) {
+    guard let arguments = call.arguments as? [Any],
+      let name = arguments.first as? String,
```

Comment:
Missing arguments should not silently fail; it's fine to force-unwrap in this case (especially since we'll convert to Pigeon shortly.